inst/tests/tests.Rraw

```diff
@@ -13955,6 +13956,18 @@ DT = data.table(
   V9 = c(0.2, -0.1, 1.2, -0.5, 1.4, 1, 0.2, 0.7, 0.4, 1.6),
   V10 = c(0.8, 0.7, -1.2, -0.9, -0.6, 0.4, -2.3, 2.2, 0.5, -1.4)
 )
+test(1971.01, DT[ , lapply(.SD, sum), .SDcols = patterns('^V', cols="V7")], error="user should not provide cols argument to patterns, when specifying the columns for melt or .SDcols; in this context, non-standard evaluation is used internally to set cols to all data table column names, so please fix by removing cols argument")
+not_patterns <- function()c("V4","V8")
+test(1971.02, DT[ , lapply(.SD, sum), .SDcols = not_patterns()], data.table(V4=3.4, V8=-0.4))
+not_patterns <- function(cols)cols
+test(1971.03, DT[ , lapply(.SD, sum), .SDcols = not_patterns()], error='argument "cols" is missing, with no default')
```

Comment:
this is just a test of base R behavior, right? if so, we should remove it -- we are just setting ourselves up to be broken if base R changes this error message in any way, and IIRC is susceptible to failing in non-English test environments as well.

if you think this test captures something from data.table, please follow the precedent of `base_messages` (circa L159) for how to test against messages "owned" by R itself

Reply:
ok I removed these tests